### ISSUE TITLE
Et Futurum Requiem crash a server after update from 2.0.2 to 2.3.1

### ISSUE THREAD

**Issue by FutureSonic-TH** [Roadhog360/Et-Futurum-Requiem#95]
I just try to update **Et Futurum Requiem** from version **2.0.2** to **2.3.1** and when I run a server after fml confirming after some time it's starting super big error spam. After 2-5 minutes of that spam, the server is crashed. I updated only this mod, so it's Et Futurum's problem. **This is not a new world if what.**

Here is the crash report - https://pastebin.com/A34cXzck

Here also a _fml-server-latest.log_ and _latest.log_. They are so big for log because It was super big console spam (more than 90 Mb log file) but if this will be helpful - here are logs, what I upload to Google Drive (because it's more than I can upload on Pastebin)

 fml-server-latest.log - https://drive.google.com/file/d/1Fu3hVPAkush2kJQ5E2cYONddOhHYQWnR/view
 latest.log - https://drive.google.com/file/d/1VRc8XcxYnNlIvo3NuiSE-7wjt6S1vY4S/view

**Comment by Roadhog360** [comment]
This might be because of boat replacement, which could also be fixed in my re-implementation of them. What happens if you disable replaceOldBoats in the entity config?

**Comment by FutureSonic-TH** [comment]
I did this and the server is loaded. It's not crashed from now. But after that when I tried to join on the server from the client - I got the error message "Fatally missing blocks and items". When I return back to version 2.0.2 from 2.3.1 - I have no problem and join normally as usual.
![Image 1](https://user-images.githubusercontent.com/40641530/147878737-6376dbd8-bd8b-426c-89dc-f2d1ed6508b0.png)
